L1Trigger/L1THGCal/python/customTriggerCellSelect.py

```diff
@@ -30,3 +34,29 @@ def custom_triggercellselect_bestchoice(process,
     parameters.NData = triggercells
     process.hgcalConcentratorProducer.ProcessorParameters = parameters
     return process
+
+
+
+
+def custom_typeenergydivision_onebitfraction(process,
```

Comment:
custom functions should now be made in a similar way as others (see for instance `custom_triggercellselect_supertriggercell`), with a clone of existing PSet and default values taken from existing PSets